File: 1470-shuffle-the-array/1470-shuffle-the-array.java
class Solution {
    public int[] shuffle(int[] nums, int n) {
        int ar[]=new int [2*n];
        int p=0;
        for(int i=0;i<2*n;i+=2)
        {
            ar[i]=nums[p];
            ar[i+1]=nums[p+n];
            p++;
        }
        return ar;
    }
}

File: 1470-shuffle-the-array/1470-shuffle-the-array.cpp
class Solution {
public:
    vector<int> shuffle(vector<int>& nums, int n) {
        vector<int> g;
        for(int i=0;i<n;i++)
        {
            g.push_back(nums[i]);
            g.push_back(nums[i+n]);
        }
        return g;
    }
};